iset.mm

```diff
@@ -81452,6 +81452,104 @@ natural numbers (notably, ~ axcaucvg ).  (Contributed by Jim Kingdon,
+      UHGVEVFVGAYTCFUFZYHUBGUFUAPUGZBSKAYOSTZUUTQQZYOUHUIZPTZYDUVDUDOZUBGUFUVAU
+      VCUVEYOUVDUKVCZUMZUVCUVBUVEUVHQZAUVBUUTVHYOUOZUPUQUVCUVFUBGUVCYDGTZQZYDUK
+      VCZUVGROUVFUVLUVMYOUVGRUVLYTUVMYOROCFUVMYPUVMYORVIAUVBUUTUVKVJUVLYDPTZUVM
+      FTZUVLUVKUVNUVOQUVCUVKVNUURUVOEYDPGUUPYDUMUUQUVMFUUPYDUKUTVAMVDUPURVKUVLU
+      VEUVHUVLUVBUVIAUVBUUTUVKVLUVJUPURVMYDUVDVOUPVPUAUBYDUVDUDPGVQVRVSAYEYCYIU
+      DOZNGUGZYIYDUDOZNGUFZXBZULUAUBPPAYCPTZUVNQZQZYEUVTUWCYEQZYCUKVCZUUAROZDFU
+      GZUUAUVMROZDFUFZXBZUVTUWDUWEUVMROZUWJUWDYEUWKUWCYEVNYCYDVOVTUWDUWEYPROZUW
+      GUUAYPROZDFUFZXBZULZUWKUWJULCSUVMYPUVMUMZUWLUWKUWOUWJYPUVMUWERWAUWQUWNUWI
+      UWGUWQUWMUWHDFYPUVMUUARWAWBWCWDUWDYQYOUUAROZDFUGZUWNXBZULZCSUFZUWPCSUFBSU
+      WEYOUWEUMZUXAUWPCSUXCYQUWLUWTUWOYOUWEYPRVIZUXCUWSUWGUWNUXCUWRUWFDFYOUWEUU
+      ARVIWEWFWDWBAUXBBSUFUWBYELWGUWDUWAUWESTZAUWAUVNYEVLYCWHZVTVKUWDUVNUVMSTAU
+      WAUVNYEVJYDWHVTVKWIUWDUWGUVQUWIUVSUWDUWFUVQDFUWDUUAFTZUWFQZQZUUAUHUIZGTZY
+      CUXJUDOZUVQUXIUXJPTZUXJUKVCZFTZUXKUXIUXMUUAUXNUMZUXIUUASTZUXMUXPQUXIAUXGU
+      XQAUWBYEUXHWJUWDUXGUWFVHZAFSUUAIWMVGUUAUOUPZUQUXIUUAUXNFUXIUXMUXPUXSURZUX
+      RUSUURUXOEUXJPGUUPUXJUMUUQUXNFUUPUXJUKUTVAMVDVBUXIUWEUXNROUXLUXIUWEUUAUXN
+      RUWDUXGUWFWKUXTVMYCUXJVOUPUVPUXLNUXJGYIUXJYCUDWAWLVRWNUWDUWIUVSUWDUWIQZUV
+      RNGUYAYIGTZQZYIUKVCZUVMROZUVRUYCUWHUYEDFUYDUUAUYDUVMRVIUWDUWIUYBWOUYCYIPT
+      ZUYDFTZUYCUYBUYFUYGQZUYAUYBVNUURUYGEYIPGUUPYIUMUUQUYDFUUPYIUKUTVAMVDZUPUR
+      VKYIYDVOUPVPXCWPWIXCWQWRAUWAYNQQZUXEUWLUEZCFUFZYPUWEROZUUCULZCSUFZUUGUYJU
+      WAUXEAUWAYNVHUXFVTAUWAYGUYLYMAUWAYGQZQZUYKCFUYQYPFTZQZUWLYCYPUHUIZUDOZUYS
+      YFVUAUEUBGUYTYDUYTUMZYEVUAYDUYTYCUDWAWSAUWAYGUYRVJAUYRUYTGTZUYPAUYRQZUYTP
+      TZUYTUKVCZFTZVUCVUDVUEYPVUFUMZVUDYPSTZVUEVUHQZAFSYPIWTYPUOZUPZUQVUDYPVUFF
+      VUDVUEVUHVULURZAUYRVNUSUURVUGEUYTPGUUPUYTUMUUQVUFFUUPUYTUKUTVAMVDVBXAVKUY
+      SUWLUWEVUFROVUAUYSYPVUFUWERAUYRVUHUYPVUMXAXDYCUYTVOXEXFVPXGAUWAYMUYOYGAUW
+      AYMQZQZUYNCSVUOVUIQZUYMUUCVUPUYMQZUYTYIUDOZNGUGZUUCVUQUYTYCUDOZVUSVUQVUFU
+      WEROVUTVUQYPVUFUWERVUQVUEVUHVUQVUIVUJVUOVUIUYMWOVUKUPZURZVUPUYMVNXHUYTYCV
+      OUPVUQYLVUTVUSULUBPUYTVUBYHVUTYKVUSYDUYTYCUDVIVUBYJVURNGYDUYTYIUDVIWEWDVU
+      OYMVUIUYMAUWAYMWKWGVUQVUEVUHVVAUQVKWIVUQVURUUCNGVUPUYMNVUOVUINAVUNNANXNUW
+      AYMNUWANXNYLNUBPNPXIYHYKNYHNXNYJNGXJXKXLXMXMVUINXNXMUYMNXNXMUUCNXNVUQUYBV
+      URUUCVUQUYBVURQZQZUYGYPUYDROZUUCVVDUYFUYGVVDUYBUYHVUQUYBVURVHUYIUPURVVDYP
+      VUFUYDRVUQVUHVVCVVBXOVVDVURVUFUYDROVUQUYBVURWKUYTYIVOVTXPUUBVVEDUYDFUUAUY
+      DYPRWAWLVRXQXRWIXCVPXSUUFUYLUYOQBUWESUXCYSUYLUUEUYOUXCYRUYKCFUXCYQUWLUXDW
+      SWBUXCUUDUYNCSUXCYTUYMUUCYOUWEYPRWAXTWBYAWLYBVS $.
+  $}
+
+  ${
+    $d A a b c d w $.  $d A a b c d x y z $.
+    $( An inhabited, bounded-above, located set of reals has a supremum.
+
+       Locatedness here means that given ` x < y ` , either there is an element
+       of ` A ` greater than ` x ` , or ` y ` is an upper bound.
```

Comment:
@jkingdon We don't know what `A` is. I'd phrase it:
```
A set of reals ` A ` is located if given any reals ... upper bound of ` A ` .
```

Reply:
I think we can just change A to "the set". I'll include that in my next pull request.

Agreed that what is there now isn't very good wording.